tests/Test.MSAL.NET.Unit/ConfidentialClientApplicationTests.cs

```diff
@@ -262,6 +264,7 @@ public void ConfidentialClientUsingCertificateTest()
             Assert.AreEqual(cachedAssertion, cc.Assertion);
 
             Assert.IsTrue(HttpMessageHandlerFactory.IsMocksQueueEmpty, "All mocks should have been consumed");
+            Assert.IsTrue(_myReceiver.EventsReceived.Count > 0);
```

Comment:
Yep. See also [this conversation](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/323/files#r111659380)